proxy/core/src/main/scala/io/cloudstate/proxy/HttpApi.scala

```diff
@@ -517,35 +523,52 @@ object HttpApi {
         )
 
       if (methDesc.isServerStreaming) {
-        def toChunk(entityMessage: MessageOrBuilder): HttpEntity.Chunk =
-          HttpEntity.Chunk(
-            if (isHttpBodyResponse) extractDataFromHttpBody(entityMessage) ++ NEWLINE_BYTES
-            else ByteString(jsonPrinter.print(entityMessage)) ++ NEWLINE_BYTES
-          )
+        val sseAccepted =
+          grpcRequest
+            .header[Accept]
+            .exists(_.mediaRanges.exists(_.value.startsWith(MediaTypes.`text/event-stream`.toString)))
```

Comment:
@raboof I think this actually shows that the akka-http API has a bit of UX work to be done ;) I've basically had to go into the akka-http code itself to figure out how to check if a request accepts a certain response type :P